bin/generate-version.js

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env node
+import { execSync } from "child_process";
+import { writeFileSync } from "fs";
+import * as https from "https";
+
+async function getLatestRelease() {
```

Comment:
I think this would be the more "node way" of doing things. But honestly I'm never a fan of just following established patterns, I always prefer what is most readable unless there is a performance issue with that.

```
#!/usr/bin/env node
#!/usr/bin/env node
import { execSync } from "child_process";
import { writeFileSync } from "fs";
import fetch from "node-fetch"; // i think fetch is the most common package to use, axios is good too

async function getLatestRelease() {
  const res = await fetch("https://api.github.com/repos/Kinto/kinto-admin/releases/latest");
  const body = await res.json();
  return body.tag_name;
}

function getGitLatestReleaseVersion() {
  try {
    return execSync("git describe --tags --abbrev=4", {
      encoding: "utf-8",
    }).trim();
  } catch (error) {
    console.log(error);
  }
};

(async() => {
  const kintoAdminVersion = process.env.KINTO_ADMIN_VERSION // coalesce until we get something
    || getGitLatestReleaseVersion() 
    || await getLatestRelease();
  writeFileSync("./public/VERSION", kintoAdminVersion);
})();
```

Note: This requires the `node-fetch` package as a dev dependency.

Reply:
> Note: This requires the node-fetch package as a dev dependency.

At first I was trying to avoid downloading `node-fetch`, but since people need to `node ci` before running `node run build` anyway, what's one more dependency?

Reply:
I try to keep dependencies slim, but when it comes to things like http clients or database clients I think they're worth it.